docs: fix broken link (#45095)
angular/angular: aio/content/cli/index.md

> Note that option names in the configuration file must use [camelCase](guide/glossary#case-types), while option names supplied to commands can use either camelCase or dash-case.

Reviewer: ```suggestion
Note that option names in the configuration file must use [camelCase](guide/glossary#case-types), while option names supplied to commands must be dash-case.
```

* See more about [Workspace Configuration](guide/workspace-config).

> * See the [complete schema](https://angular.io/guide/workspace-config#angular-workspace-configuration) for `angular.json`.

Reviewer: The link here, is exactly the same as the one above. I think we can drop this whole point.

> * See the [complete schema](https://angular.io/guide/workspace-config#angular-workspace-configuration) for `angular.json`.

Reviewer: ```suggestion

```
